### PR TITLE
Add image pull secret

### PR DESCRIPTION
**Pull request by stone-z**
## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Comment by stone-z** [comment]
Tested on `gaia`, will merge after https://github.com/giantswarm/config/pull/805

**Comment by stone-z** [comment]
Config has been [released](https://github.com/giantswarm/config/releases/tag/v1.589.0), just waiting on team review

**Review by fiunchinho — APPROVED** [review]

**Pull request merged**
